RNS/__init__.py

```diff
@@ -58,6 +59,17 @@
 LOG_DEBUG    = 6
 LOG_EXTREME  = 7
 
+_log_map_to_standard = (
+    logging.CRITICAL,
+    logging.ERROR,
+    logging.WARNING,
+    25,  # NOTICE
+    logging.INFO,
+    15,  # VERBOSE
+    logging.DEBUG,
+    5,  # EXTREME
+    )
+
```

Comment:
This might be a bit pedantic, but I'd like to see `NOTICE`, `VERBOSE`, and `EXTREME` defined in this scope for easy, verbose comparisons later. I'm also curious if it would be possible to change the values of these levels, as in `logging`, the higher the priority of the level, the higher the number. Whereas here we do the reverse, and this might end up causing some confusion if we were to at some point base some behavior off of the highness of the log level value.

Adding as a comment rather than a review as I might just be silly.